test/harness/index.js

```diff
@@ -140,7 +151,7 @@ function assert_invalid(bytes) {
 const assert_malformed = assert_invalid;
 
 function assert_soft_invalid(bytes) {
-    test(() => {
+    uniqueTest(() => {
```

Comment:
I liked the consistency with the WPT `test` function, which takes arguments in the same order. I don't have a strong opinion though, so I am alright with changing it, if you strongly prefer it.